contracts/tg4-mixer/Cargo.toml

```diff
@@ -38,16 +37,14 @@ tg4 = { path = "../../packages/tg4", version = "0.10.0" }
 tg-utils = { path = "../../packages/utils", version = "0.10.0" }
 tg-bindings = { path = "../../packages/bindings", version = "0.10.0" }
 
-# bench dependencies
```

Comment:
Better

Reply:
Oh, I moved that to another PR for clarity: https://github.com/confio/poe-contracts/pull/152.

Reply:
Thanks for the review.